src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -91,7 +97,30 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
 
         pop = self.initialize_medication_coverage(pop)
 
+        # Generate multiplier columns
+        pop[data_values.COLUMNS.SBP_MULTIPLIER] = 1
+        mask_sbp_adherent = (
+            pop[data_values.COLUMNS.SBP_MEDICATION_ADHERENCE]
+            == data_values.MEDICATION_ADHERENCE_TYPE.ADHERENT
+        )
+        mask_sbp_one_drug = (
+            pop[data_values.COLUMNS.SBP_MEDICATION]
+            == data_values.SBP_MEDICATION_LEVEL.ONE_DRUG_HALF_DOSE.DESCRIPTION
+        )
+        mask_sbp_two_drugs = (
+            pop[data_values.COLUMNS.SBP_MEDICATION]
+            == data_values.SBP_MEDICATION_LEVEL.TWO_DRUGS_HALF_DOSE.DESCRIPTION
+        )
+        pop.loc[
+            (mask_sbp_adherent) & (mask_sbp_one_drug), data_values.COLUMNS.SBP_MULTIPLIER
+        ] = data_values.SBP_MULTIPLIER.ONE_DRUG
+        pop.loc[
+            (mask_sbp_adherent) & (mask_sbp_two_drugs), data_values.COLUMNS.SBP_MULTIPLIER
+        ] = data_values.SBP_MULTIPLIER.TWO_DRUGS
```

Comment:
I like having the column name in data values.  It makes it easier to reference instead of having to type the string every time.

Reply:
Yeah, i don't think data_values.py is really where it belongs, but def containerizing column names is a useful thing